data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+				goto questions
+
+			label "been to hope"
+			`	"Oh, wow! It must have been a harrowing experience. Adelita does a powerful job of giving you the whole story, though, and I bet you'll still learn a lot. You might even feel as if you've been transported back there yourself," she says with a mischievous grin. "You know, this was their first work that really got them famous in the interplanetary scene, but they've been forced to close it and today's their final day. I had to see it one more time before they went."`
+
+			label questions
+			choice
+				`	"Who's Adelita?"`
+				`	"Why did they have to close it?"`
+					goto signage
+
+			`	"They're an independent journalistic collective whose goals are to bring the injustices of human civilization to light. No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their use of mixed media, their eye for story telling... I just love it. I know you're going to love it too," she says.`
+				goto exhibition
+
+			label signage
+			`	"Well, this exhibition used to be permanent, and it garnered a lot of attention, but the planetary government was pressured by the Paradise Planets to force Adelita to leave, because it gave such a candid view of what they did to those poor people. Personally, I think what Parliament and the Paradise Planets have been doing to the people of Hope and their advocates like Adelita has just been abhorrent and wrong."`
+
+			label exhibition
+			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews, and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them are quite crude and tasteless. It's a strong rebuke to see those displayed here, so close to the Paradise Planets.`
+			`	There are many, many photos, but one in particular catches your eye. It's of a woman standing in line to board a ship, a baby held at her hip and a look of sadness and determination on her face. The mother and child's cheeks, rosy and chapped, stand out in contrast with the cold blues in the background.`
+			`	You head towards the exit, but as the automatic doors open, an arctic blast pummels into you from out of nowhere. Instead of <origin>'s spaceport, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```

Comment:
```suggestion
			`	You head toward the exit, but as the automatic doors open, an arctic blast pummels into you from out of nowhere. Instead of <origin>'s spaceport, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```